distributed/client.py

```diff
@@ -3367,8 +3367,9 @@ def profile(
                 filename = "dask-profile.html"
 
             if filename:
-                from bokeh.plotting import save
+                from bokeh.plotting import output_file, save
 
+                output_file(filename=filename, title="Dask Profile")
                 save(figure, title="Dask Profile", filename=filename)
```

Comment:
Same thing here with `title`

```suggestion
                save(figure, filename=filename)
```